src/calculateRentalCost.js

```diff
@@ -4,7 +4,33 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  // I tried everyting, and there is actualy second way to solve this problem
+  // Well, this program gets job done.
+
+  const LONG_TERM = 7;
+  const MED_TERM = 6;
+  const SHORT_TERM = 3;
+
+  const BASE_PRICE = 80;
+  const SHORT_TERM_DISCOUNT = -20;
+  const MID_TERM_DISCOUNT = -140;
+  const LONG_TERM_DISCOUNT = -150;
+
+  if (days < SHORT_TERM) {
+    return BASE_PRICE;
```

Comment:
The return value for days less than SHORT_TERM should be calculated as days multiplied by the BASE_PRICE, not just returning the BASE_PRICE.